tests/cupy_tests/lib_tests/test_polynomial.py

```diff
@@ -694,7 +694,7 @@ def test_polyfit_weighted_diff_types(self, xp, dtype1, dtype2, dtype3):
 class TestPolyval(Poly1dTestBase):
 
     @testing.for_all_dtypes()
-    @testing.numpy_cupy_allclose(rtol={numpy.float16: 1e-2, 'default': 1e-3})
+    @testing.numpy_cupy_allclose(rtol={numpy.float16: 1e-2, 'default': 2e-3})
```

Comment:
This changes tolerance for both rocm and cuda, please add a conditional check for rocm. You can use runtime.is_hip to check the platform.

Reply:
Hey @pnunna93 Added this check. UT passes
[ut_lib.log](https://github.com/user-attachments/files/17635543/ut_lib.log)

Reply:
Thanks, please fix file permissions and static checks.